### PR TITLE
[Layout] Add Id to the container div

### PR DESCRIPTION
**Pull request by StevenTCramer**
# Pull Request

## 📖 Description

The Id property was not being set on FluentLayout

### 🎫 Issues

Was trying to scope css using Id and noticed it was missing on this component.

## 👩‍💻 Reviewer Notes

Oh yea I fixed the version warning.

```
microsoft\fluentui-blazor\tests\Core\Microsoft.FluentUI.AspNetCore.Components.Tests.csproj : warning NU1603: Microsoft.FluentUI.AspNetCore.Components.Tests depends on FluentAssertions (>= 7.0.0-
tags-7.0.0-tags-7-0-0-alpha-3.1) but FluentAssertions 7.0.0-tags-7.0.0-tags-7-0-0-alpha-3.1 was not found. An approximate best match of FluentAssertions 7.0.0-tags-7-0-0-alpha-3.1 was resolved.
```

## 📑 Test Plan

Added FluentLayout_Id Test to ensure Id attribute renders

### General

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fluentui-blazor/blob/master/docs/contributing.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have added [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for my new compontent
- [x] I have modified an existing component
- [x] I have validated the [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for an existing component

### CODE REVIEW

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: Directory.Packages.props [hunk 1]
```diff
@@ -31,11 +31,11 @@
     <PackageVersion Include="System.Text.Json" Version="8.0.3" />
     <!-- Test dependencies -->
     <PackageVersion Include="bunit" Version="1.28.9" />
-    <PackageVersion Include="FluentAssertions" Version="7.0.0-alpha.3" />
+    <PackageVersion Include="FluentAssertions" Version="7.0.0-tags-7.0.0-tags-7-0-0-alpha-3.1" />
```
[the remark this answers is not in the document]
Comment (StevenTCramer) [reply Directory.Packages.props:34]: When I run tests I get the warning and I only changed it to what it was resolving to anyway.  
I have no idea why alpha version was selected.  Vincent updated to alpha 3 and I went back as far as history goes and the earlies version is alpha 2.  I was simply resolving the warning.

[the remark this answers is not in the document]
Comment (StevenTCramer) [reply Directory.Packages.props:34]: They must be coming from Azure Dev Ops.
```
╰─ﬀ cat .\nuget.config
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
  </packageSources>
</configuration>
╭─   D:\git\github\microsoft\fluentui-blazor   Cramer/2024-05-29/FluentLayoutId ≢  羽8ms⠀                                                                       ﴃ CIC-Service-Center ﴱ docker-desktop
╰─ﬀ dotnet nuget list source
Registered Sources:
  1.  dotnet-public [Enabled]
      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json
      

```

[the remark this answers is not in the document]
Comment (StevenTCramer) [reply Directory.Packages.props:34]: Yep

image

[the remark this answers is not in the document]
Comment (vnbaaij) [reply Directory.Packages.props:34]: Yeah, I think I had issues with the stable release not being available in the `dotnet-public` feed. 

As we only use this package for testing and the test project compiles and runs with these pre-release versions, I'm ok with those versions.

**Review by vnbaaij — APPROVED** [review]

File: Directory.Packages.props (hunk 1 above)
[the remark this answers is not in the document]
Comment (dvoituron) [reply Directory.Packages.props:34]: OK. I will update the `nuget.config` file later. Not an issue at this moment. Thanks

**Review by dvoituron — APPROVED** [review]

**Pull request merged**